src/Catel.MVVM/Catel.MVVM.Shared/MVVM/Commands/Command.cs

```diff
@@ -244,7 +244,7 @@ public void Execute(object parameter)
         public void Execute(TExecuteParameter parameter)
         {
 #pragma warning disable 4014
-            ExecuteAsync(parameter, false);
+            ExecuteAsync(parameter, false).Wait();
```

Comment:
Imho an async execution of the command has not be done. 
When I am understanding it right: if the command would be async, the user could press e.g. the button again and again if the execution would be a long running task?
And as I am not able to await the execution ... this would be a horrible scenario to me ;-)